src/jaxoplanet/experimental/starry/rotation.py

```diff
@@ -201,12 +201,10 @@ def dotR(l_max: int, u: Array) -> Callable[[Array], Array]:
         - theta is the rotation angle in radians
     """
     Rls = [Rl(l) for l in range(l_max + 1)]
-    n_max = l_max**2 + 2 * l_max + 1
 
-    @partial(jnp.vectorize, signature=f"({n_max},{n_max}),()->({n_max},{n_max})")
```

Comment:
@lgrcia — This call to `vectorize` was wrong and (I think) unnecessary. I think it's possible that _all_ the `vectorize` calls are similarly incorrect. Let's investigate.